### ISSUE TITLE
Ops file use-external-dbs.yml not playing well with latest bosh director?

### ISSUE THREAD

**Issue by aleliaert** [cloudfoundry/cf-deployment#534]
Seeing this from our bosh deploy when using cf-deployment against a director made with latest bosh-deployment (director 266.2.0): 
```
Error: Job 'locket' in instance group 'diego-api' specifies consumers in the manifest but the job does not define any consumers in the release spec
```
When looking in [use-external-dbs.yml](https://github.com/cloudfoundry/cf-deployment/blob/master/operations/use-external-dbs.yml), it is doing a `replace` on several jobs' `consumes` content with `{datbase: nil}`.  Is this approach to removing the internal database relationship not compatible with newer directors?

Note that we managed to work around the the issue by changing the same ops file to perform `remove` on `consumes?/database`.

Can submit a PR with our workaround.  Thanks!

**Comment by jmcarp** [comment]
Should be fixed once #525 is released.

**Comment by aleliaert** [comment]
Thanks!  Will close this as it's a duplicate.
